src/core/Group.js

```diff
@@ -1807,7 +1807,10 @@ Phaser.Group.prototype.preUpdate = function ()
     {
         var child = this.children[i];
 
-        child.preUpdate();
+        if (child.active)
```

Comment:
So right now groups have the pendingDestroy check before this and do the renderOrderID calls even when inactive. I get that this removes the function call overhead, but should we not let the children decide whether or not to update in their own update rather than the parent doing it?

Reply:
Good point.

Reply:
Now preUpdate function calls won't be blocked by parents, and all preUpdate calls will call their coreUpdate even if they are not active so that they can destroy themselves again.  **preUpdate** functions will now consider their parents as well.

**postUpdate**, **update** and **render** calls will all prune child calls through the active variable.